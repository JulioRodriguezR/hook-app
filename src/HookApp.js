import React from 'react'

const bgrContent = {
    backgroundColor: 'darkgray',
    borderBottom: '10px',
    width: '32%'
};

export const HookApp = () => {
    return (
        <div className='row'>
            <div className='col-4' style={bgrContent}>
                <h1 className='p-3'>Hook App</h1>
            </div>
            <div className='col-8'></div>
        </div>
    )
}
