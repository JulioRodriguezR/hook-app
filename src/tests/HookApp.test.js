import React from 'react';
import { shallow } from 'enzyme';

import { HookApp } from '../HookApp';

describe('Test in <HookApp />', () => {
  
  test('Should show correctly', () => {
   const wrapper = shallow(<HookApp />);
   expect(wrapper).toMatchSnapshot(); 
  });
  
});