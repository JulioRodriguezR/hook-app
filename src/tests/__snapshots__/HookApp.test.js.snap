// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <HookApp /> Should show correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-4"
    style={
      Object {
        "backgroundColor": "darkgray",
        "borderBottom": "10px",
        "width": "32%",
      }
    }
  >
    <h1
      className="p-3"
    >
      Hook App
    </h1>
  </div>
  <div
    className="col-8"
  />
</div>
`;
